### PR TITLE
refactor: use 'number' instead of 'string' in data objects

### PR DESCRIPTION
**Pull request by tbouffard**
The related data is actually a number, so the objects should reflect this.

### Notes

The value passed as overlay label property is transformed into a String. This is a current limitation of bpmn-visualization.
If the transform is not done, an error occurs. I will fill an issue in the bpmn-visualization repository. The label type is string in the API types but nothing prevents the JavaScript code to pass a number.

In the following stack trace build.js is the mxGraph code!
```
Uncaught TypeError: str.split is not a function
    plainText build.js:20625
    text build.js:20472
    paint build.js:26794
    redrawShape build.js:22994
    redraw build.js:22831
    redraw build.js:26853
    redrawCellOverlays build.js:50100
    visit build.js:1432
    redrawCellOverlays build.js:50074
    redraw build.js:50319
    addCellOverlay build.js:57438
    addOverlays bpmn-visualization.esm.js:5467
    addOverlays bpmn-visualization.esm.js:5465
    addOverlays bpmn-visualization.esm.js:5794
```

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -35,10 +35,10 @@ let edgesWaitingInstances = getEdgesWaitingInstances();
 // Add Overlay on activities
 activitiesMonitoringData.forEach((value, key) => {
   // running on time
-  if (value.onTime != false) {
+  if (value.onTime) {
     bpmnVisualization.bpmnElementsRegistry.addOverlays(key, {
       position: "top-center",
-      label: value.onTime,
+      label: String(value.onTime),
```
Comment (tbouffard) [inline src/index.js:41]: If you prefer, we could use instead a [string template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)
```suggestion
      label: `${value.onTime}`,
```

Comment (csouchet) [reply src/index.js:41]: I prefer the suggestion of @tbouffard.

Comment (tbouffard) [reply src/index.js:41]: Well, the survey is
- @assynour `String`
- @csouchet string template
- @tbouffard I don't care!

😵‍💫

Comment (tbouffard) [reply src/index.js:41]: decision taken collectively: use string template

**Review by csouchet — APPROVED** [review]

**Pull request merged**